pom.xml

```diff
@@ -255,6 +255,17 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
Could you try removing this to see if the forked copy of commons-validator that is bundled into Jenkins will also work?

See https://github.com/jenkinsci/jenkins/tree/stable-2.138/core/src/main/java/jenkins/org/apache/commons/validator/routines for the forked copy inside Jenkins so that the plugin does not need to bring in another copy of commons-validator.

Reply:
Yes, it feels strange to me that there is a forked copy inside Jenkins, but it seems like it is there...

Reply:
Yes I used the forked copy initially. It is has a @Restricted(NoExternalUse.class). 
So the build was failing because of this particular import, hence I used the external commons-validator for it.

Reply:
Thanks for checking. The no external use restriction is a good thing. It tells us that you made exactly the right choice to explicitly depend on the official commons-validator.